api/lib/domain/services/authentication/oidc-authentication-service.js

```diff
@@ -81,7 +82,8 @@ class OidcAuthenticationService {
     }
 
     const isEnabledInConfig = config[this.configKey].isEnabled;
-    if (!isEnabledInConfig) {
+    const isEnabledInPixAdmin = config[this.configKey].isEnabledInPixAdmin;
+    if (!isEnabledInConfig && !isEnabledInPixAdmin) {
```

Comment:
Il y a un nommage `isReadyForPixAdmin` et puis un nommage `isEnabledInPixAdmin`, donc une fois c'est « For » et une autre fois c'est « In ». Ce n'est pas très cohérent.

De plus la propriété de configuration `isEnabledInPixAdmin` est vraiment très spécifique, alors que le service, lui, est générique avec la notion d'audience. Alors pourquoi ne pas plutôt avoir une propriété `audiences` qui prendrait une liste ?

```javascript
 google: {
      isEnabled: false,
      audiences: getArrayOfStrings(process.env.GOOGLE_AUDIENCES),
 }
 ```

Reply:
On s'est posé beaucoup de questions sur la bonne manière de définir les critères d'affichage d'un SSO (par application, par nom de domaine, par extension, par locale…) et on a pensé qu'il était trop tôt pour prendre une décision (d'autant que Pix Admin n'est pas concerné au-delà du choix de l'app). On a donc choisi de faire au plus simple pour l'instant, avec une variable d'environnement qui disparaîtra de toute façon prochainement.

Reply:
Bien reçu. Est-ce qu'on peut alors au moins unifier entre le « For » et le « In ».